lib/core/src/Cardano/Wallet/Api/Server.hs

```diff
@@ -5043,23 +5043,12 @@ instance IsServerError ErrAssignRedeemers where
                 , "for one of your redeemers since I am unable to decode it"
                 , "into a valid Plutus data:", pretty r <> "."
                 ]
-        ErrAssignRedeemersUnresolvedTxIns ins ->
-            -- Note that although this error is thrown from
-            -- '_assignScriptRedeemers', it's more related to balanceTransaction
-            -- in general than to assigning redeemers. Hence we don't mention
-            -- redeemers in the message.
-            apiError err400 UnresolvedInputs $ T.unwords
-                [ "The transaction I was given contains inputs I don't know"
-                , "about. Please ensure all foreign inputs are specified as "
-                , "part of the API request. The unknown inputs are:\n\n"
-                , pretty ins
-                ]
-        ErrAssignRedeemersTranslationError TimeTranslationPastHorizon ->
```

Comment:
Probably `TranslationLogicMissingInput` is the replacement for the removed `UnknownTxIns` error, so:

Suggestion:
```haskell
        ErrAssignRedeemersTranslationError (TranslationLogicMissingInput inp) ->
             -- Note that although this error is thrown from
             -- '_assignScriptRedeemers', it's more related to balanceTransaction
             -- in general than to assigning redeemers. Hence we don't mention
             -- redeemers in the message.
             apiError err400 UnresolvedInputs $ T.unwords
                 [ "The transaction I was given contains inputs I don't know"
                 , "about. Please ensure all foreign inputs are specified as "
                 , "part of the API request. The unknown input is:\n\n"
                 , T.pack $ show inp
                 ]
```
then we should be able to keep the test (and this error code) unchanged.

Reply:
For some reason the test coverage seems very poor now. Not sure what's changed. I think it might be best to mark it pending / remove, and if anything have one or two simple unit tests.

I pushed to a separate branch, feel free to cherry-pick:
- a7a446c51e78a806e07b65e0306ab824f44ee934
- b4c2ec474e8803e7cbc58d0e46568fce627fe654

Reply:
Thank you, wanted to ask you about this :+1: